include/strings.h

```diff
@@ -11,15 +11,6 @@
 
 #define LMAX 256
 
-#ifndef NULL
-#define NULL 0		       /* null pointer constant */
-#endif
-
-#ifndef _SIZE_T
-#define _SIZE_T
-typedef unsigned int size_t;  /* result type of the sizeof operator (ANSI) */
-#endif
```

Comment:
Later in this file (as I can't comment somewhere as low), we may want to change this line:

```
#define strdup(p)	( strcpy(malloc(strlen(p)+1),p); )
```
to
```
#define strdup(p)	do { strcpy(malloc(strlen(p)+1),p); } while(0)
```